packages/firebase_auth/firebase_auth_web/lib/firebase_auth_web.dart

```diff
@@ -402,7 +402,11 @@ class FirebaseAuthWeb extends FirebaseAuthPlatform {
       // origin as a single string
       delegate.useAuthEmulator('http://$host:$port');
     } catch (e) {
-      throw getFirebaseAuthException(e);
+      final String code = (e as dynamic).code;
+      // this catches Firebase Error from web that occurs after hot reloading & hot restarting
+      if (code != 'auth/emulator-config-failed') {
+        throw getFirebaseAuthException(e);
```

Comment:
I think we said during meeting that a log should be displayed here so user aren't surprised